.github/scripts/ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
This script should/can be used for all other arches too... while dropping the actions/haskell/setup thing